### ISSUE TITLE
Tflite-micro supported layers

### ISSUE THREAD

**Issue by panhu** [tensorflow/tflite-micro#1820]
Hi:
I want to ask if there are BatchNormalization layers in my model, does tflite-micro support?
